Makefile

```
@@ -1391,6 +1391,10 @@ $(GOVULNCHECK): # Build govulncheck.
 $(IMPORT_BOSS): # Build import-boss
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) $(IMPORT_BOSS_PKG) $(IMPORT_BOSS_BIN) $(IMPORT_BOSS_VER)
 
+.PHONY: provider-issues
+provider-issues: # Creates GitHub issues in a pre-defined list of CAPI provider repositories
+	@go run ./hack/tools/release/internal/update_providers/provider_issues.go
```

```suggestion
.PHONY: release-provider-issues-tool
provider-issues: # Creates GitHub issues in a pre-defined list of CAPI provider repositories
	@go run ./hack/tools/release/internal/update_providers/provider_issues.go
```

Let's make this a build target like the release notes tool and let's put it beside that target in the Makefile.

---

I am not sure I understood this right. Did you mean to move this make target around the release-notes-tool targets after updating the target name? If yes, I have done so. If otherwise, please lemme know.